control-plane-pki-operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -91,8 +91,6 @@ func NewCertRotationController(
 			Client:        kubeClient.CoreV1(),
 			EventRecorder: eventRecorder,
 			Owner:         ownerRef,
-			JiraComponent: "HOSTEDCP",
```

Comment:
Did they remove these fields?

Reply:
but now i see it is possible to use `AdditionalAnnotations` to put these values in.
i'm not sure how to do it properly, as i needed to fix the code so it will compile to unblock CI.
you might want to open a followup PR to get them back in. sorry for inconvenience.

Reply:
fixing in https://github.com/openshift/hypershift/pull/4070